vault/resource_consul_secret_backend_role_test.go

```diff
@@ -157,20 +170,24 @@ resource "vault_consul_secret_backend_role" "test" {
   partition = "partition-0"
 `, path, token, name)
 
-	if withPolicies {
+	if usePolicies {
 		config += `
-  policies = [
+policies = [
     "boo",
-  ]
+]
 `
 	}
 
-	if isAboveVersionThreshold {
+	if useCPolicies {
```

Comment:
Do we really need to two booleans for this? I think we could do it in an`if/else` instead

Reply:
I don't think that will work because of the combination of values used to test things. Some tests use both at the same time. I don't see any way to limit it down to 2 booleans current.

I think re-writing the test into 2 separate test functions like we discussed will make it able to be simplified a little bit, though.